nautobot/core/urls.py

```diff
@@ -45,6 +45,8 @@
     path("", include("social_django.urls", namespace="social")),
     # django-health-check
     path(r"health/", include("health_check.urls")),
+    # FileProxy attachments
+    path("files/", include("db_file_storage.urls")),
```

Comment:
Not being familiar with db_file_storage, what views does this add? Are they useful, do we need to document them somewhere and/or add them to a nav menu somewhere?

Reply:
Just basic views URLS for retrieving/downloading the files. I had to add them for the admin integration to be able to have detail views for the files.